Source/Core/InputCommon/GCAdapter.cpp

```diff
@@ -266,7 +535,23 @@ void StopScanThread()
 
 static void Setup()
 {
-	libusb_device** list;
+#if MEASURE_POLLS_FROM_ENGINE
+	__time64_t long_time;
+	_time64(&long_time);
+	struct tm newtime;
+	_localtime64_s(&newtime, &long_time);
```

Comment:
Linux doesn't have a `__time64_t`, and has no 64-bit timer on 32-bit Kernel, you need to use `time_t` and call
`localtime_r(&long_time, &newtime)` (args are switched).